navbar.js

```diff
@@ -1,38 +1,38 @@
-var categoriesSort = categories.sort((a, b) => {
+var categoriesSort = categories.sort(function(a, b) {
   return b.count - a.count;
 });
 
-var tmp = 0;
+var categoryFiveCounter = 0;
```

Comment:
i think this counter is used only in one place, isn't it? it seems like it can be overridden by another function which would introduce bugs to expected result. i propose to only declare it here (not giving it value, `var zmienna;`) and then rebuild the function using it to something like this
```
var firstFiveCategory = function(){
 categoryFiveCounter = 0;
 categoriesSort.filter(function(category) {
 categoryFiveCounter++;
    return categoryFiveCounter <= 5;
 });
};
```
so now you can call firstFiveCategory method more than once